sympy/__init__.py

```diff
@@ -22,6 +22,21 @@
 
 from sympy.release import __version__
 
+if 'dev' in __version__:
+    def enable_warnings():
+        import os
+        import warnings
+        action ='default'
+        travis = (os.environ.get('TRAVIS','') == 'true')
+        if travis:
+            action = 'error'
+        warnings.filterwarnings(action,   '.*',                      DeprecationWarning, module='sympy.*')
+        # let's not fail the test on that yet. 
```

Comment:
I mean there's no way to avoid the warning while supporting Python 2.7. Although I suppose we could do different things in 2 and 3.